src/Components/PresentationalComponents/CSAwRuleBox/CSAwRuleBox.js

```diff
@@ -184,27 +178,22 @@ const CSAwRuleBox = ({ rules, synopsis, changeExposedSystemsParameters, intl })
                                                 </Label>
                                             </SplitItem>
                                             <SplitItem isFilled className="icon-with-label">
-                                                { RISK_OF_CHANGE_LABEL[rule.change_risk] }
+                                                { RISK_OF_CHANGE_LABEL[4] }
```

Comment:
I think this is a mistake, now every rule appears to have "high" risk of change.

Reply:
phew, great catch! I was testing the labels. will update